CTP/shell/src/com/navercorp/cubridqa/shell/common/CommonUtils.java

```diff
@@ -120,12 +124,20 @@ public static String concatFile(String p1, String p2) {
 		return p.replace('/', File.separatorChar);
 	}
 	
-	public static String getFileNameForDispatchAll(){
-		return CommonUtils.concatFile(Constants.DIR_CONF, "dispatch_tc_ALL.txt");
+	public static String getFileNameForDispatchAll(Context context){
+		return CommonUtils.concatFile(context.getToolHome() + "/" + Constants.CURRENT_LOG_DIR, "dispatch_tc_ALL.txt");
 	}
 
-	public static String getFileNameForDispatchFin(String envId){
-		return CommonUtils.concatFile(Constants.DIR_CONF,"dispatch_tc_FIN_" + envId + ".txt");
+	public static String getFileNameForDispatchFin(String envId, Context context){
+		return CommonUtils.concatFile(context.getToolHome() + "/" + Constants.CURRENT_LOG_DIR,"dispatch_tc_FIN_" + envId + ".txt");
+	}
+	
+	public static String getEnvInMemory(String key){
```

Comment:
It's better remove getEnvInMemory().

Just use com.navercorp.cubridqa.common.CommonUtils.getEnvInFile(key) as required.